src/components/redirects/redirects-routes.ts

```diff
@@ -2,6 +2,7 @@ import * as express from "express";
 import { Request, Response } from "express";
 import { PATH_DATA } from "../../app.constants";
 import { requiresAuthMiddleware } from "../../middleware/requires-auth-middleware";
+import { WELL_KNOWN_FILES } from "../../app.constants";
```

Comment:
code smell ;-)
```suggestion
import { PATH_DATA, WELL_KNOWN_FILES } from "../../app.constants";
```

Reply:
oh good spot, I'll refactor that now...
🤔 I thought the linter would be complaining about that? How'd I get past CI?

Reply:
Great that's amended now, thanks @TaherKapasi!

I'll take a look at the lintier rules... pretty sure it used to pull me up on those!